.gitignore

```diff
@@ -21,7 +21,7 @@ Thumbs.db
 
 coverage.txt
 
-/internal/api/e2e/test/
+/internal/api/e2e/api/test/
```

Comment:
sounds weird we need that here

Reply:
Well it's where json test files from e2e api integration tests are currently created 😄